substrate-ingest/src/parse/util.ts

```diff
@@ -28,6 +28,13 @@ export function formatId(height: number, hash: string, index?: number): string {
 }
 
 
+export function formatEvmLogId(height: number, index: number): string {
```

Comment:
Why do we need block hash in log id?
`{height}-{log_idx}` format helps me do fast queries like:
```sql
SELECT id FROM evm_log WHERE contract = $1 AND id > $2;
```
and avoid using offset

Reply:
1. This query is not fast, because it will pick either a `contract` index or `id`, but in most cases `(contract, id)` index would perform much better.
2. Including a hash into `id` doesn't break this assumption.

The overall idea of including a hash is questionable, but it allows us in theory to store unfinilized blocks as well. Since we include a block hash everywhere, it is reasonable to include it into `evm_log` as well.

Reply:
> This query is not fast, because it will pick either a contract index or id, but in most cases (contract, id) index would perform much better.

I disagree with it. I haven't checked how it works in postgres but for cockroach the idea works fine:
query - `select id from evm_log where contract = '0x0000000000000000000100000000000000000084' order by id;`
index - `CREATE INDEX IDX_evm_log__contract ON evm_log(contract);`
execution plan:
```
karura_twin=> explain analyze select id from evm_log where contract = '0x0000000000000000000100000000000000000084' order by id;
                                                   info                                                   
----------------------------------------------------------------------------------------------------------
 planning time: 498µs
 execution time: 5ms
 distribution: local
 vectorized: true
 rows read from KV: 5,953 (541 KiB)
 cumulative time spent in KV: 4ms
 maximum memory usage: 680 KiB
 network usage: 0 B (0 messages)
 
 • scan
   nodes: n5
   actual row count: 5,953
   KV time: 4ms
   KV contention time: 0µs
   KV rows read: 5,953
   KV bytes read: 541 KiB
   estimated max memory allocated: 680 KiB
   estimated row count: 5,967 (30% of the table; stats collected 20 hours ago)
   table: evm_log@idx_evm_log__contract
   spans: [/'0x0000000000000000000100000000000000000084' - /'0x0000000000000000000100000000000000000084']
(20 rows)
```
I don't see any downsides

Reply:
> Including a hash into id doesn't break ordering by height.

Including `hash` and `event_idx` between `height` and `log_idx` make me to use offset which is unwanted